modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/jaxrs/exception/mapper/DuplicateProductSpecificationSpecificationKeyExceptionMapper.java

```diff
@@ -23,12 +23,12 @@
 	property = {
 		"osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Catalog)",
 		"osgi.jaxrs.extension=true",
-		"osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Catalog.DuplicateProductSpecificationKeyExceptionMapper"
+		"osgi.jaxrs.name=Liferay.Headless.Commerce.Admin.Catalog.DuplicateProductSpecificationSpecificationKeyExceptionMapper"
```

Comment:
This looks like a typo, why did you doubled _Specification_?

Reply:
Because the field is called `specificationKey` and the object `ProductSpecification`. I did this because now we also have the `key` field and this was confusing